content/zh-cn/docs/concepts/storage/volumes.md

```diff
@@ -788,19 +788,39 @@ root 身份运行进程，或者修改主机上的文件权限，以便能够从
 -->
 #### hostPath 配置示例
 
+{{< tabs name="hostpath_examples" >}}
+
 <!--
-Linux node
+{{< tab name="Linux 节点" codelang="yaml" >}}
```

Comment:
No. This doesn't work.

<img width="595" alt="image" src="https://github.com/user-attachments/assets/15d68901-799d-48ba-8bbe-645ae3f9f4f2">